frontend/src/components/about/Member.js

```diff
@@ -1,7 +1,35 @@
 import React from 'react'
+import Header from './Header'
+import MemberInfo from './MemberInfo'
+import './about.css'
+
 
 function Member() {
-    return <div>-member component placeholder-</div>
+    const IMG_URI = "https://previews.123rf.com/images/studiostoks/studiostoks1604/studiostoks160400296/55726621-male-business-coach-draws-on-background-pop-art-retro-style-business-training-drawing-board.jpg";
+    const LINK_IMAGE = "https://cdn-icons-png.flaticon.com/512/38/38669.png";
+    const LINK = "https://uofablueprint.org"
+
+    return (
+        <>
+            <Header text="Meet the Team"/>
+
+            <div className='container'>
```

Comment:
I think the layout looks great, we just need to make it dynamic so that we can display any number of MemberInfo component

Reply:
Typically, the Member component will get an array of object containing member data and we iterate through this array to display each MemberInfo component which will received the member data as a prop from each object in the array.

Helpful link: https://stackabuse.com/how-to-loop-in-react-jsx/

Reply:
For now, we can store the members data in a local json file. You can just create members.json in src/data. But we'll be replacing with data from the database in the coming sprints